app/helpers/arclight_helper.rb

```diff
@@ -43,6 +43,14 @@ def fields_have_content?(document, field_accessor)
     end
   end
 
+  # If we have a facet on the repository, then return the Repository object for it
+  #
+  # @return [Repository]
+  def repository_faceted_on
+    repos = facets_from_request.find { |f| f.name == 'repository_sim' }.try(:items)
+    Arclight::Repository.find_by(name: repos.first.value) if repos.length == 1 && repos.first.value
```

Comment:
If `try` fails here `repos` will be `nil`.  But we call `length` and `first` on repos on this line.  Could this be potentially problematic if we don't find a repository?